packages/g/gd-sprest-bs.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "gd-sprest-bs",
+  "filename": "dist/gd-sprest-bs-icons.min.js",
```

```suggestion
  "filename": "gd-sprest-bs-icons.min.js",
```
As the auto-update config uses a base of `dist`, we don't need it here.